layouts/shortcodes/logs_support_table.md

```diff
@@ -0,0 +1,15 @@
+{{ $data := $.Site.Data.instrumentation.languages }}
```

Comment:
Use kebab case for the shortcode file name.

Also, there is probably a way to generate the table content automatically, but this is fine for now.

Reply:
Good call out! I didn't do it to be consistent with the existing `telemetry_support_table.md`, maybe we merge this one and then we fix them in a go (metrics_support_table.md should be in now as well)

> Also, there is probably a way to generate the table content automatically, but this is fine for now.

I tried that with `range` but failed terrible 😆

Reply:
Will merge this one and then create a follow up PR with kebab case